IsraelHiking.Web/src/scss/dialogs.scss

```diff
@@ -58,4 +58,10 @@
         min-width: 48px;
         height: 48px;
     }
+}
+
+@media(max-height:550px){
+    #dialogContentForScroll {
```

Comment:
Is it possible to use something more generic?

Reply:
Yes we can use class name instead of id, but the i don't know that default class might me used in other modal and cause unwanted results. Or we can add new class name on these 2 modals.

Reply:
This issue applies to all the modals as far as I know. 
Much like the title changes, this should be applied to all the modals.

Reply:
I think the "mat-mdc-dialog-container" addition in this file were an early attempt to fix a similar issue for mobile screens...

Reply:
Sure, i will update and replace id with class.